index.html

```diff
@@ -274,7 +274,7 @@ <h3>Contact me</h3>
           <textarea name="message" id="usr-comment" class="textarea" maxlength="500" placeholder="Write your message here" required></textarea>
```

Comment:
[ OPTIONAL ] Please add unnecessary files and folders like `.vscode` to `.gitignore`.